pkg/logs/docker/docker_test.go

```diff
@@ -0,0 +1,39 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package docker
+
+import (
+	"errors"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseMessage(t *testing.T) {
+	preMessage := "100000002018-06-14T18:27:03.246999277Z "
+	longLog := []byte(preMessage + strings.Repeat("a", 16*1024) + preMessage + strings.Repeat("b", 50))
+	msgLength := len(strings.Repeat("a", 16*1024) + strings.Repeat("b", 50))
+	ts, status, msg, err := ParseMessage(longLog)
+	assert.NotNil(t, ts)
+	assert.NotNil(t, status)
+	assert.NotNil(t, msg)
+	assert.Nil(t, err)
+	assert.Equal(t, msgLength, len(msg))
```

Comment:
I think you check the content of all the different values returned by the method.

Reply:
@ajacquemot Is that a bad thing?  Should I only test `err` and `msg`?

Reply:
I don’t think it’s a bad thing but not testing the content of `msg` can be.